libs/application/templates/public-debt-payment-plan/src/fields/Overview/DistributionTabel.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { PaymentDistribution } from '../../lib/dataSchema'
+import { useDistributionTable } from '../../lib/paymentPlanUtils'
+import { PaymentPlanTable } from '../components/PaymentPlanTable/PaymentPlanTable'
+
+export const DistributionTable = ({
+  monthAmount,
+  monthCount,
+  totalAmount,
+  scheduleType,
+}: PaymentDistribution) => {
+  const { isLoading, distributionData } = useDistributionTable({
+    monthAmount,
+    monthCount,
+    totalAmount,
+    scheduleType,
+  })
+  if (!distributionData) return null
```

Comment:
Do we need to display something to the client in this case?

Reply:
Maybe, but I think we need to rethink errors later. There are so many questions I have regarding this form, that I think many things will change later regarding errors.